### PR TITLE
Store work.

### PR DESCRIPTION
**Pull request by runer53248**

### CODE REVIEW

File: shm/Store.cpp [hunk 1]
```diff
@@ -1,40 +1,40 @@
...
+    if (!cargo) {
         return Response::lack_of_cargo;
     }
-    if(!player){
+    if (!player) {
         return Response::lack_of_cargo;
     }
-    if(!amount){
+    if (!amount) {
         return Response::lack_of_cargo;
```
Comment (Martyna-Tu) [inline shm/Store.cpp:5]: I think that you can catch these 3 "if conditions" to 1
and do the same in sell method

File: shm/Cargo.hpp [hunk 2]
```diff
@@ -18,5 +19,16 @@ class Cargo {
...
+    }
 
+    std::unique_ptr<Cargo> split(size_t amountPart) {
+        if (!amountPart or amountPart > amount_) {
+            return {};
+        }
+
+        *this -= amountPart;
+        return std::make_unique<Cargo>(name_, amountPart, basePrice_);
+    }
```
Comment (Rogal255) [inline shm/Cargo.hpp]: Is there a reason to put implementation of this method in header file?

**Review by Rogal255 — APPROVED** [review]
OK for me. Don't forget to track changes in our main "shm" branch.

File: shm/Store.cpp (hunk 1 above)
Comment (runer53248) [reply shm/Store.cpp:5]: Done.

File: shm/Cargo.hpp (hunk 2 above)
Comment (runer53248) [reply shm/Cargo.hpp]: Moved implementation to cpp.

**Pull request merged**